dnfdragora/ui.py

```diff
@@ -1072,9 +1073,9 @@ def _setInfoOnWidget(self, pkg) :
                     if pkg.updateinfo :
                         s += '<b>%s</b>'%pkg.updateinfo[0]['title']
                         s += "<br>"
-                        s += pkg.updateinfo[0]['description'].replace("\n", "<br>")
+                        s += escape(pkg.updateinfo[0]['description'])
```

Comment:
Same as above for description append 
`.replace("\n", "<br>")`

Reply:
OK